src/core/repoIdRegistry.test.js

```diff
@@ -9,6 +9,7 @@ import {
 } from "./repoIdRegistry";
 
 import {makeRepoId} from "../core/repoId";
+import {fakeTimestamp} from "../cli/load.test";
```

Comment:
src/core should never depend on src/cli. if we needed a fake timestamp export, it should be exported from core and then imported in src/cli. but exporting a variable that is just a fake timestamp is unnecessary, I think--each file can define its own fake timestamp

Reply:
This sounds like a much better idea to me as well. Trying to go about this in a practical sense was very difficult for me, testing is still a foreign concept to me at this point. I'll fix this up though and perhaps we can find a better solution later on.

I watched a presentation from the Jest team the other night [https://youtu.be/cAKYQpTC7MA](url), they were discussing the new release(v23) of Jest and some of it's new features. At 42:47 in the video one of the devs begin to talk about using mock dates with a feature called snapshot property matchers, this sounds like it might be relevant and useful. I definitely need to look more into this today.